### ISSUE TITLE
Small feature: character limit information

### ISSUE THREAD

**Issue by srtalbot** [cds-snc/platform-forms-client#606]
**User story**: Persona based
As an end-user
I need to know how many characters I have remaining in a textArea
When there is a character limit on the field
So that I know how much more information I am able to provide

**Story context**
Information thanks to Shawn Thompson at AAACT: 

If we are using a max character for the field, we should be giving users additional information, like a character counter: see the USA's design pattern for information on how to make these accessible: https://designsystem.digital.gov/components/character-count/

This is another good one he saw: https://github.com/rikschennink/short-and-sweet

Here is an example form: https://forms-staging.cdssandbox.xyz/id/234

Applicable WCAG criteria:
- Success Criterion [3.3.3 Error Suggestion](https://www.w3.org/TR/WCAG21/#input-assistance)

**Acceptance criteria**
- [ ] character limit information provides equal service in French and English.
- [ ] character limit information works on Chrome, Firefox, Edge, Safari, and Opera.
- [ ] An error message informs users that they have too many characters and the correct number of characters are check on validation.
- [ ] character limit information is read out by a screen reader.
- [ ] character limit information observe colour contrast requirements. 
- [ ] manual accessibility testing meet WCAG 2.1 AA requirements.
